SPECS-SIGNED/mft_kernel-signed/mft_kernel-signed.spec

```
@@ -5,14 +5,12 @@
 
 %global KVERSION %{target_kernel_version_full}
 
-%global _name kernel-mft
-
-Name:		 %{_name}
-Summary:	 %{name} Kernel Module for the %{KVERSION} kernel
-Version:	 4.30.0
-Release:	 1%{?dist}
-License:	 Dual BSD/GPLv2
-Group:		 System Environment/Kernel
+Name:            mft_kernel
+Summary:         %{name} Kernel Module for the %{KVERSION} kernel
```

Suggestion: I'm guessing you may want this to be consistent with the naming:
```suggestion
Summary:         kernel-mft Kernel Module for the %{KVERSION} kernel
```